.github/workflows/run-entry-points.yml

```
@@ -27,3 +27,5 @@ jobs:
       run: |
         yaqd-mcc-118 --version
         yaqd-mcc-118 --help
+	yaqd-mcc-152 --version
+	yaqd-mcc-152 --help
```

```suggestion
        yaqd-mcc-152 --version
        yaqd-mcc-152 --help
```

This is a tabs vs spaces thing, it probably looked correct in your editor, but consistency is key here